app/views/lead_providers/guidance/npq_usage.html.erb

```diff
@@ -85,7 +78,74 @@
 </p>
 
 
-<h2 id="declaring-that-an-npq-participant-has-started-their-induction" class="govuk-heading-l">
+<h2 id="make-a-declaration" class="govuk-heading-l">Making a declaration for an NPQ participant</h2>
+
+<h3 class="govuk-heading-m">
+  What is a declaration?
+</h3>
+
+<p class="govuk-body-m">
+  A declaration is a message in JSON format sent to the participant-declarations API.
+</p>
+
+<p class="govuk-body-m">
+  This will allow providers’ own software systems to integrate with our DfE service - these declarations will not need to be made manually by provider data teams.
+</p>
+
+<p class="govuk-body-m">
+  A participant refers to a late career teacher undertaking a given NPQ course on a given provider learning platform.
+</p>
+
+<p class="govuk-body-m">
+  For a participant on a course, a declaration is the formal communication from a lead provider to the department that a training event for a specific participant has happened and can be evidenced on the providers’ platform.
+</p>
+
+<p class="govuk-body-m">
+  For example, participant Joe Bloggs who started their induction/course on a lead provider platform requires the lead provider to submit a <code>started</code> declaration for Joe Bloggs to the department.
+</p>
+
+<h3 class="govuk-heading-m">
+  Declaration submission?
+</h3>
+
+<p class="govuk-body-m">
+  Declarations provide contractual MI reporting requirements for lead providers.
+</p>
+
+<p class="govuk-body-m">
+  They are the sole means for triggering a milestone payment to a lead provider for a participant.
+</p>
+
+<p class="govuk-body-m">
+  They are submitted in line with the contractual milestones set-out by the DfE.
+</p>
+
+<p class="govuk-body-m">
+  The participant-declarations API will accept the declarations required to trigger payments.
+</p>
+
+<p class="govuk-body-m">
+  The validations process means declarations make sense in relation to the relevant milestone dates and the declarations previously submitted for the participant (available to test link maybe)
+</p>
+
+<p class="govuk-body-m">
+  Invalid declarations will be excluded from the payment calculation. They will be rejected with a specific reason with most rejections being asynchronous initially.
+</p>
+
+<p class="govuk-body-m">
+  In exceptional circumstances where declarations cannot be evidenced on time and thus cannot be submitted on time; the API will accept backdated declarations.The late declaration will need to be backdated appropriately to fall within the relevant milestone period.
```

Comment:
`declarations.The late declaration will need` - need a space after the dot

Reply:
Good spot.